app/Http/Controllers/BotController.php

```diff
@@ -33,9 +33,15 @@ public function destroy(Bot $bot)
         try {
             $this->botRepository->delete($bot->id);
 
-            return redirect('/bots')->with('messageSuccess', __('message.bot.notification.delete.success'));
+            return redirect('/bots')->with('messageSuccess', [
+                'status' => 'Delete success',
+                'message' => __('message.bot.notification.delete.success')
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/BotController.php</br>**Line**: 38</br>Each line in an array declaration must end in a comma